.travis.yml

```diff
@@ -13,9 +13,7 @@ before_install:
 install:
     - conda create -n testenv --yes pip python=$PYTHON_VERSION
     - source activate testenv
-    - source install_requirements.sh
-    - pip install codecov
-    - pip install -q flake8
+    - pip install -r testing-requirements.txt
```

Comment:
since we are using conda envs, let's install with conda here?

Reply:
What is the command ? I don't use conda..

Reply:
ok, bad idea. If a single library is not installable by conda the whole thing crashes.

Reply:
Maybe use an `environement.yml` file then (example [here](https://github.com/binder-project/example-conda-environment/blob/master/environment.yml)) that can be used to update an existing conda environment and can specify both conda and pip packages ? 

Also as a side note, I find https://github.com/astropy/ci-helpers very useful for setting up continuous integration.

Reply:
Which one is not installable? All in that list are for me installable with conda

Reply:
Ah, but that is probably because I have conda-forge channel added. OK, then leave it be.

Regarding jupyter, what is this used for? Is it only for nbconvert? (then can also only install that)

Reply:
@rth  How do you use an `environment.yml` to specify pip packages ?

IMO `ci-helpers` is quite a big machinery for pure python packages. However we may use it in the future, especially for the very simple way to define build matrices.

Reply:
Yes, jupyter is only for nbconvert, I only call it in the backend to generate the page on ramp.studio. So it should be tested in the travis of the kit, but not in rampwf.

Reply:
@aboucaud Well one could do, `conda env create [or update] -f envonement.yml` where, `environement.yml` looks like, e.g.,
```yml
name: test-env
channels:
  - defaults
  - conda-forge
dependencies:
  - python=3.5
  - numpy=1.13.1
  - scipy=0.19.1
  - conda_package_1
  - conda_package_2
  - pip :
    - pip_package_1 
    - ./  # install the current package
```
but in the end it's not that different from just running conda install and pip install separately..

Reply:
Seems convenient indeed. I'll keep that in mind.